citi-onboarding/virtu-onepage
# update(linkedin-profile):

###What i did
- updated linkedin profile on linkedin section and took out console.logs

###How to test
1. Execute the command `yarn` to download the server modules
2. Execute the command `cd server`
3. Execute the command `node index.js` to start the server
4. Execute the command `cd ../client` and then `yarn` to download the client modules
5. Execute the command `yarn start` to start the client
6. Open another browser page on `http://localhost:3000` to see the client page
7. See if the changes I made are working
